desc/objectives/_coils.py

```diff
@@ -850,11 +850,15 @@ def body(k):
         return min_dist_per_coil
 
 
-class PlasmaCoilSetMinDistance(_Objective):
-    """Target the minimum distance between the plasma and coilset.
+class PlasmaCoilSetDistanceBound(_Objective):
+    """Target the distance between the plasma and coilset.
 
-    Will yield one value per coil in the coilset, which is the minimum distance from
-    that coil to the plasma boundary surface.
+    Will yield one or two values per coil in the coilset, depending on the mode
```

Comment:
Should mention somewhere here that the minima is the first N values in the array, then the maxima, it's not interleaved